cartography/cli.py

```diff
@@ -403,6 +403,23 @@ def _build_parser(self):
                 'The crowdstrike URL, if using self-hosted. Defaults to the public crowdstrike API URL otherwise.'
             ),
         )
+        parser.add_argument(
+            '--gsuite-auth-method',
+            type=str,
+            default='delegated',
+            choices=['delegated', 'oauth'],
+            help=(
+                'The method used by GSuite to authenticate. delegated is the legacy one.'
+            ),
+        )
+        parser.add_argument(
+            '--gsuite-tokens-env-var',
+            type=str,
+            default='GSUITE_GOOGLE_APPLICATION_CREDENTIALS',
+            help=(
+                'The method used by GSuite to authenticate. delegated is the legacy one.'
```

Comment:
Crap, I missed one thing in my review actually - what should the docstring here be? This is the same text message for `--gsuite-auth-method`.

Sorry for the back and forth.